portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,9 +28,41 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> comments;
```

Comment:
nit: prefer to have references to interfaces instead of concrete classes (`java.util.List` in this case). Preferably, the most generic one that still has the functionality you need.

Reply:
Fixed